packages/design-system/src/components/OcDatepicker/OcDatepicker.vue

```diff
@@ -1,145 +1,113 @@
 <template>
-  <date-picker ref="datePicker" class="oc-datepicker" v-bind="$attrs" :popover="popperOpts">
-    <template #default="args">
-      <!-- @slot Default slot to use as the popover anchor for datepicker -->
-      <!-- args is undefined during initial render, hence we check it here -->
-      <slot
-        v-if="args"
-        :input-value="args.inputValue"
-        :toggle-popover="args.togglePopover"
-        :hide-popover="args.hidePopover"
-      />
-    </template>
-  </date-picker>
+  <oc-text-input
+    v-model="dateInputString"
+    v-bind="$attrs"
+    :label="label"
+    type="date"
+    :min="minDate?.toISODate()"
+    :fix-message-line="true"
+    :error-message="errorMessage"
+    :clear-button-enabled="true"
+    :clear-button-accessible-label="$gettext('Clear date')"
+    class="oc-date-picker"
+  />
 </template>
 
 <script lang="ts">
-import {
-  computed,
-  ComponentPublicInstance,
-  defineComponent,
-  defineAsyncComponent,
-  ref,
-  unref,
-  watch
-} from 'vue'
-import { Modifier } from '@popperjs/core'
-import 'v-calendar/dist/style.css'
-import OcSpinner from '../OcSpinner/OcSpinner.vue'
+import { computed, defineComponent, onMounted, PropType, ref, unref, watch } from 'vue'
+import { useGettext } from 'vue3-gettext'
+import { DateTime } from 'luxon'
 
-/**
- * Datepicker component based on [v-calendar](https://vcalendar.io/). For detailed documentation, please visit https://vcalendar.io/vue-3.html
- */
 export default defineComponent({
   name: 'OcDatepicker',
   status: 'ready',
   release: '1.0.0',
+  props: {
+    label: { type: String, required: true },
+    currentDate: { type: Object as PropType<DateTime>, required: false, default: null },
+    minDate: { type: Object as PropType<DateTime>, required: false, default: null }
+  },
+  emits: ['dateChanged'],
+  setup(props, { emit }) {
+    const { $gettext, current } = useGettext()
+    const dateInputString = ref<string>('')
 
-  components: {
-    DatePicker: defineAsyncComponent({
-      loader: async () => {
-        const { DatePicker } = await import('v-calendar')
-        return DatePicker
-      },
-      loadingComponent: OcSpinner
+    const date = computed(() => {
+      const date = DateTime.fromISO(unref(dateInputString)).endOf('day')
+      return date.isValid ? date : null
     })
-  },
 
-  inheritAttrs: true,
-  setup() {
-    const datePicker = ref<ComponentPublicInstance>()
+    const minDateUndercut = computed(() => {
```

Comment:
What do you think about

```suggestion
    const isMinDateUndercut = computed(() => {
```

to make it clearer that it's a boolean? 🤔